src/ecosystem/Ecosystem.jl

```diff
@@ -74,40 +74,44 @@ end
+    iv__seed_fog_shade__wave_stress::P = Param(1.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
         name="Wave Stress", description="Importance of avoiding wave stress. Higher values places more weight on areas with low wave stress.")
-    heat_stress::P = Param(1.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
+    iv__seed_fog_shade__heat_stress::P = Param(1.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
         name="Heat Stress", description="Importance of avoiding heat stress. Higher values places more weight on areas with low heat stress.")
-    shade_connectivity::P = Param(0.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
+    iv__fog_shade__in_connectivity::P = Param(0.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
         name="Shade Connectivity", description="Higher values give preference to locations with high connectivity for shading deployments.")
-    in_seed_connectivity::P = Param(1.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
-        name="Incoming Connectivity (Seed)", description="Higher values give preference to locations with high incoming connectivity (i.e., receives larvae from other sites) for enhanced coral deployments.")
-    out_seed_connectivity::P = Param(1.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
-        name="Outgoing Connectivity (Seed)", description="Higher values give preference to locations with high outgoing connectivity (i.e., provides larvae to other sites) for enhanced coral deployments.")
-    coral_cover_low::P = Param(0.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
-        name="Low Coral Cover", description="Higher values give greater preference to sites with low coral cover for seeding deployments.")
-    coral_cover_high::P = Param(0.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
-        name="High Coral Cover", description="Higher values give preference to sites with high coral cover for shading deployments.")
-    seed_priority::P = Param(1.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
-        name="Predecessor Priority (Seed)", description="Importance of seeding sites that provide larvae to priority reefs.")
-    shade_priority::P = Param(0.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
-        name="Predecessor Priority (Shade)", description="Importance of shading sites that provide larvae to priority reefs.")
-    zone_seed::P = Param(0.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
-        name="Zone Predecessor (Seed)", description="Importance of seeding sites that provide larvae to priority (target) zones.")
-    zone_shade::P = Param(0.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
-        name="Zone Predecessor (Shade)", description="Importance of shading sites that provide larvae to priority (target) zones.")
-    coral_cover_tol::P = Param(0.2, ptype="real", bounds=(0.0, 1.0), dists="unif",
+    iv__seed__in_connectivity::P = Param(1.0, ptype="real", bounds=(0.0, 1.0), dists="unif",
```

Comment:
Descriptions seem to have been duplicated and no longer relate to what the parameters represent

Reply:
I  may have missed updating some, I can change this.